docs/Gemfile.lock

```diff
@@ -0,0 +1,73 @@
+GEM
```

Comment:
@attilapiros, thanks for working on this. How did you create this file out of curiosity?

Reply:
You are welcome! I bumped into this problem in the spark-website where it is really annoying because of the diffs it generates. 

As it is given in the description:
> docs/Gemfile.lock is generated by the "bundle install".